esp-hal/src/rtc_cntl/mod.rs

```diff
@@ -424,9 +414,9 @@ impl RtcClock {
                     .ck8m_force_pu()
                     .bit(matches!(slow_freq, RtcSlowClock::RtcSlowClock8mD256))
             });
-
-            ets_delay_us(300u32);
         };
+
+        crate::rom::ets_delay_us(300u32);
```

Comment:
```suggestion
        crate::rom::ets_delay_us(300);
```
It does not add any value unless Im missing something.